.changeset/tasty-parents-chew.md

```diff
@@ -0,0 +1,7 @@
+---
+'@sveltejs/site-kit': patch
+---
+
+forward searchbox slot
+
+pick 2175c9d79 Apply 1 suggestion(s) to 1 file(s)
```

Comment:
```suggestion
forward searchbox slot
```

Reply:
For some reason I can't make that change myself - could you remove the "pick .." thingy? Then we're good to merge